CHANGELOG.md

```diff
@@ -1,10 +1,24 @@
 # Release History
-## 0.9.0 (Unreleased)
+## 0.9.0 (2022-08-31)
 
 ### New Features:
 - Added support for displaying source code as comments in the generated scripts when registering UDFs.
 This feature is turned on by default. To turn if off, pass the new keyword argument `source_code_display` as `False` when calling `register()` or `@udf()`.
+- Added support for `column_names` in `df.write.save_as_table()`. You can now specify the names of columns when appending new data to an existing table.
+- Added function `get_active_session()` in module `snowflake.snowpark.context` to get the current active Snowpark session.
+- Added keyword argument `overwrite` to `session.write_pandas()` to allow overwriting contents of a Snowflake table with that of a Pandas DataFrame.
+- Added `FileOperation.put_stream()` to upload local files to a stage via file stream.
```

Comment:
I would have all of these start with "Added support for" so that the wording is consistent

Reply:
I feel "added support for" might not be the best for all cases if we're simply adding adding argument/method/functions.

But I got your point on consistency, I have tweaked a bit to make them consistent, how do you like it now?